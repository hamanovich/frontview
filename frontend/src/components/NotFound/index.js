import React from 'react';
import FontAwesome from 'react-fontawesome';
import { Link } from 'react-router-dom';

import Jumbotron from 'react-bootstrap/Jumbotron';
import Button from 'react-bootstrap/Button';

const NotFound = () => (
  <Jumbotron className="text-center text-danger">
    <h1 className="display-3">
      <FontAwesome name="exclamation-triangle" />
      <br />
      404: not Found
    </h1>
    <Link to="/">
      <Button variant="success" size="lg">
        Return to home page
      </Button>
    </Link>
  </Jumbotron>
);

export default NotFound;
