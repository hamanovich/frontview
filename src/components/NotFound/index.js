import React from 'react';
import FontAwesome from 'react-fontawesome';

import Jumbotron from 'react-bootstrap/lib/Jumbotron';

const NotFound = () => (
  <Jumbotron className="text-center">
    <h1>
      <FontAwesome name="exclamation-triangle" />
      <br />
      <span className="text-danger">404: not Found</span>
    </h1>
  </Jumbotron>
);

export default NotFound;
