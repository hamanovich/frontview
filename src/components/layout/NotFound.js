import React from 'react';

import Jumbotron from 'react-bootstrap/lib/Jumbotron';

const NotFound = () => (
  <Jumbotron className="text-center">
    <h1>404: Page not Found</h1>
  </Jumbotron>
);

export default NotFound;
