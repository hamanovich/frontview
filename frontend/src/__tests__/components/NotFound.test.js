import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../components/NotFound';

describe('<NotFound/>', () => {
  it('renders <NotFound /> component', () => {
    const component = shallow(<NotFound />);

    expect(component).toMatchSnapshot();
  });
});
