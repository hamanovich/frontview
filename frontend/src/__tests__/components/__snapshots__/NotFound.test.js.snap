// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound/> renders <NotFound /> component 1`] = `
<Bootstrap(Jumbotron)
  className="text-center text-danger"
>
  <h1
    className="display-3"
  >
    <FontAwesome
      name="exclamation-triangle"
    />
    <br />
    404: not Found
  </h1>
  <Link
    to="/"
  >
    <Button
      active={false}
      disabled={false}
      size="lg"
      type="button"
      variant="success"
    >
      Return to home page
    </Button>
  </Link>
</Bootstrap(Jumbotron)>
`;
